github/teams.go

```diff
@@ -355,3 +355,122 @@ func (s *TeamsService) ListUserTeams(ctx context.Context, opt *ListOptions) ([]*
 
 	return teams, resp, nil
 }
+
+type ListTeamProjectsOptions struct {
```

Comment:
I'm thinking this is a copy-paste error.

I don't see any mention of `Affiliation` in the description of this API endpoint:
https://developer.github.com/v3/teams/#list-team-projects

Please delete the entire struct and remove it as an option from the following API endpoint.